src/components/AdminNicety.js

```diff
@@ -33,10 +39,25 @@ const AdminNicety = React.createClass({
     },
 
     textareaChange: function() {
-        this.setState({ text: event.target.value , noSave: false });
+        this.setState({ text: event.target.value, noSave: false });
     },
 
     render: function() {
+        let reviewedRender;
+        if (this.state.reviewedValue === true) {
+          reviewedRender = (
+            <Checkbox checked onChange={this.reviewedChange}>
+            Reviewed
+            </Checkbox>
+          );
+        } else if (this.state.reviewedValue === false) {
+          reviewedRender = (
+            <Checkbox onChange={this.reviewedChange}>
+            Reviewed
+            </Checkbox>
+          );
+        }
```

Comment:
React offers a better way! Boolean props, like `checked`, can either be implicit as above (present => true, absent => false) or explicit, eg `checked={true}`. That allows this to be both simplified and inlined:

```jsx
<Checkbox
  checked={this.state.reviewedValue === true}
  onChange={this.reviewedChange}
>
  Reviewed
</Checkbox>
```